test/unit/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepositorySpec.scala

```diff
@@ -48,15 +49,14 @@ class CaseRepositorySpec extends BaseMongoIndexSpec
 
   private val conflict = 11000
 
-  private val mongoDbProvider = new MongoDbProvider {
+  private val mongoDbProvider: MongoDbProvider = new MongoDbProvider {
```

Comment:
type not required for private val.   (you don't have to change - just showing that I'm noticing things(

Reply:
Intellij was complaining anyway hence why I did it